src/mscorlib/src/System/String.cs

```diff
@@ -411,7 +415,61 @@ private unsafe static bool EqualsHelper(String strA, String strB)
                 return (length <= 0);
             }
         }
-        
+
+        [System.Security.SecuritySafeCritical]  // auto-generated
+        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
+        private unsafe static bool OrdinalCompareSubstring(String strA, String strB, int length)
```

Comment:
Sure thing. I based the name off the CoreRT version of EqualsHelper, OrdinalCompareEqualLengthStrings.